samples/vs-code/covid-19.dotnet-interactive

```diff
@@ -118,7 +118,7 @@ notebookScope.plot = function (plotTarget) {
                         name: "Recovered",
                         type: "scattergeo",
                         mode: "markers",
-                        geo: "geo1",
```

Comment:
Do we want to make these same changes in the equivalent `.ipynb`?

Reply:
let me do that

Reply:
done